conf.json

```diff
@@ -460,6 +460,11 @@
 				"descr": "When timers are raised on elections, report a warning if the background task holds the lock longer than this value.",
 				"def": "100ms", "min": 0, "max": "max" },
 
+			{ "type": "bool", "name": "sqliterepo_allow_master",
+				"key": "sqliterepo.election.allow_master",
+				"descr": "Allow the role of MASTER in any election.",
+				"def": false },
```

Comment:
I think it should be true by default.

Reply:
`false` is the worst default value that was possible.